src/ex1/index.html

```diff
@@ -1,14 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <title>Exercise 1</title>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ori's Todo list 📝</title>
     <link rel="stylesheet" href="style.css" />
-</head>
-<body>
 
-<h1>Todo list</h1>
+    <script
+      src="https://kit.fontawesome.com/361581c500.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+  
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/bodymovin/5.9.4/lottie.min.js"
+      integrity="sha512-ilxj730331yM7NbrJAICVJcRmPFErDqQhXJcn+PLbkXdE031JJbcK87Wt4VbAK+YY6/67L+N8p7KdzGoaRjsTg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    ></script>
```

Comment:
<img width="1414" alt="image" src="https://user-images.githubusercontent.com/96736630/169696677-bfd392c5-d571-4a4b-b449-9d2dde2e8f18.png">
Pay attention that adding the above scripts for animations & font increased the code size for your application by 9 times! (lottie + font = 64Kb, your script (8kb)

Achieving the confetti and delete animations could be done using regular css so this is a bit overkill for such a small exercise,  but as you already did it - at least it was done with a min.js file which is good and the use of the lottie animation function made properly.